pkgs/dart_services/lib/src/compiling.dart

```diff
@@ -136,12 +136,23 @@ class Compiler {
 
       Directory(path.join(temp.path, 'lib')).createSync(recursive: true);
 
-      final bootstrapPath = path.join(temp.path, 'lib', kBootstrapDart);
-      final bootstrapContents =
-          usingFlutter ? kBootstrapFlutterCode : kBootstrapDartCode;
+      final mainPath = path.join(temp.path, 'lib', kMainDart);
+      File(mainPath).writeAsStringSync(source);
 
-      File(bootstrapPath).writeAsStringSync(bootstrapContents);
-      File(path.join(temp.path, 'lib', kMainDart)).writeAsStringSync(source);
+      String compilePath;
+      if (useNew && deltaDill != null) {
```

Comment:
It's separate from this PR, be can you clarify what '`useNew`' is used for? Perhaps either document it in the `_compileDDC()` dartdocs, or change the parameter name to clarify what it is (`useHotReloadCodepath`? `useNewModules`?).